package.json

```diff
@@ -14,7 +14,6 @@
   },
   "author": "dxw",
   "license": "MIT",
-  "dependencies": {},
```

Comment:
I assume this was unintentional as I get a `MODULE_NOT_FOUND` error when trying to run the server.

Reply:
yep, good spot. Some rebase issue.